design/one-pager-managed-resource-api-design.md

```diff
@@ -51,12 +51,27 @@ deviate so that we can reassess and improve it.
 
 ## Guiding Principles
 
-Our users are the stakeholders on how we make decisions. There are two types of users for managed resources:
-* Input Users: Users who configure/create/manage the resource. `Crossplane Claim` Controller, `Cluster Admin` are a few
-  examples of this type. We want our `Spec` to be the source of truth for them to configure everything about the resource.
-* Output Users: Users who refer to/query/use/extract information from the managed resource. We want them to be able to
-  perform all read-only operations on the given resource as if they have the cloud provider's API available. This can
-  include configurations and metadata information about the resource.
+Our users are the stakeholders on how we make decisions. There are two
+types of users for managed resources:
+* Input Users: Users who configure/create/manage the resource. We want
+  our `Spec` to be the source of truth for them to configure everything
+  about the resource.
+* Output Users: Users who refer to/query/use/extract information from
+  the managed resource. We want them to be able to perform all read-only
+  operations on the given resource as if they have the cloud provider's
+  API available. This can include configurations and metadata
+  information about the resource.
+
+### Group Version Kind (GVK)
+
+All Kubernetes Custom Resources belong to a group and are of a certain
+kind and version. There may be multiple kinds in a group and multiple
+versions of the same kind. In Crossplane, Custom Resources should mirror
+the provider API type that they represent as close as possible in group
+and kind. For instance, AWS has a `Cluster` type that is part of the
+`eks` [package]. The first version of the `Cluster` custom resource
+should be of `kind: Cluster` and `apiVersion:
+eks.aws.crossplane.io/v1alpha1`.
   
 ### Naming Conventions
```

Comment:
@muvaf we should probably update this section after https://github.com/crossplane/crossplane/pull/1493 is resolved